src/component/projects.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Styled from 'styled-components';
+import Gallery from 'react-photo-gallery';
+
+
+const Container = Styled.div`
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  padding-top: 3vw;
+  padding-bottom: 3vw;
+`;
+
+const FullThemeline = Styled.div`
```

Comment:
ThemeLine is a common component but this is specific to the page and the design is different from that previous one